app/src/main/java/org/wikipedia/WikipediaApp.java

```diff
@@ -301,6 +301,13 @@ public boolean setFontSizeMultiplier(int multiplier) {
         return false;
     }
 
+    public void setFontFamily(@NonNull String fontFamily) {
+        if (!fontFamily.equals(Prefs.getFontFamily())) {
+            Prefs.setFontFamily(fontFamily);
+            bus.post(new ThemeChangeEvent());
```

Comment:
May be we should rename it to something more generic like ThemeFontChangeEvent() or AppearanceChangeEvent()?

Reply:
Sounds good. I've updated to `ThemeFontChangeEvent()`.